mp/src/game/client/momentum/ui/gameui/mainmenu/MainMenu.cpp

```diff
@@ -309,6 +315,8 @@ void MainMenu::DrawMainMenu()
         }
     }
 
+    SetPostProcessParams(&m_PostProcessParameters, true);
```

Comment:
Should this move a few lines down to inside the `IsInLevel` check? That's where it was previously.

Reply:
I figured that won't be necessary because we're moving to model panel for the menu background, but better safe than sorry I suppose.